packages/@react-stately/slider/src/useSliderState.ts

```diff
@@ -110,6 +110,11 @@ export function useSliderState(props: SliderProps): SliderState {
     // Round value to multiple of step, clamp value between min and max
     value = clamp(getRoundedValue(value), thisMin, thisMax);
 
+    // Do nothing if slider hasn't moved
```

Comment:
ah... because `replaceIndex` always generates a new array even though it has the same values in it

would it be better to put this into the `replaceIndex` function and have `replaceIndex` return the same array if the values all end up the same?

otherwise anyplace that ever uses `replaceIndex` will need to be aware of this change

Reply:
Even if the `replaceIndex` is fixed to return the same array, it will still trigger the `onChange` and `onChangeEnd` props.

Reply:
that is true, but we could test for same array instead, which is more in line with how everything else determines state changes

Reply:
I feel that `replaceIndex` should be consistent in the way that it always returns a different array. Thus instead of calling it and comparing array objects to skip further actions, it's better to just not do anything as soon as we find out that slider value hasn't changed.

Reply:
👍 i'm good with this